Website/Packages/datax-metrics/src/modules/metrics/components/metricExplorer.jsx

```diff
@@ -129,8 +129,7 @@ const contentStyle = {
     width: '100%',
     display: 'flex',
     flexDirection: 'column',
-    overflowY: 'auto',
-    paddingBottom: 80
```

Comment:
i think we want to keep 80 and not 20 tho.  Is the effect the same?

Reply:
Yes I don't see any difference. And just for consistency, I decided to keep 20.